Additions: 3; Deletions: 5; Total Changes: 8
--- test.rs
+++ test2.rs
@@ -1,19 +1,17 @@
 use std::fs::File;
 
 use flate2::Compression;
-/* Multi
-    line
-        comment
-            will be removed*/
+
 use flate2::write::GZEncoder;
 // Same comment
+
 fn main() -> Result<(), std::io::Error>{
 
 let gz = File::create("archive.tar.gz"); // heere is a comment
 
 let encoder = GZEncoder::new(gz, Compression::default());
 
-let mut tar = tar::Builder::new(enc);
+let mut tar232 = tar::Builder::new(enc);
 
 // add all files in the current directory to current_backup
 
